instanceidhandler/v1/helpers/keys.go

```diff
@@ -49,11 +50,15 @@ const (
 
 // Statuses
 const (
-	Initializing = "initializing"
-	Ready        = "ready"
-	Completed    = "completed"
-	Incomplete   = "incomplete"
-	Unauthorize  = "unauthorize"
+	Initializing   = "initializing"
+	Ready          = "ready"
+	Completed      = "completed"
+	Incomplete     = "incomplete"
+	Unauthorize    = "unauthorize"
+	Partial        = "partial"
```

Comment:
Partial and full are not statuses, they are Completion modes.
Maybe its better to have a different const block for the two modes to prevent confusion.
And I think the values should be:
`partial`/`complete`